apps/minifront/src/state/ibc-in/constants.ts

```diff
@@ -0,0 +1,7 @@
+// Chains using specifically bech32 encoding (not bech32m)
+export const bech32Chains = ['noble', 'nobletestnet'];
+
+export const blockExplorerTxBase: Record<string, string> = {
+  'osmo-test-5': 'https://www.mintscan.io/osmosis-testnet/tx',
+  'grand-1': 'https://www.mintscan.io/noble-testnet/tx',
```

Comment:
we can get it from the cosmos registry
https://github.com/cosmos/chain-registry/blob/2e7e7f67d58c71cad1bc9d82d9342b6e48a72026/noble/chain.json#L234-L257

Reply:
Ah! That is indeed better. Going to revise this PR to use this.